components/mpas-ocean/cime_config/config_compsets.xml

```diff
@@ -9,18 +9,43 @@
 
   <!-- E3SM C/G compsets -->
 
+  <compset>
+    <alias>CMPASO-S-ISMF</alias>
+    <lname>2000_SATM_SLND_SICE_MPASO%ISMFDATMFORCED_SROF_SGLC_SWAV</lname>
+    <support_level>"Experimental, under development"</support_level>
+  </compset>
```

Comment:
This one doesn't work (yet...).

Reply:
This is the stub atmosphere case?

Reply:
Yes, this is the stub atmosphere.  I'll take this out if I can't get it working.